### PR TITLE
Create official Documentation

### PR DESCRIPTION
**Pull request by flagbird**
https://flagbit.github.io/category-bundle is currently linked to the documentation branch. It will be changed to the main branch once it's merged.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+## Category Bundle
+
+The Flagbit Akeneo Category Bundle allows you to add new category properties like text-fields, checkboxes, date-fields and
+many more to enhance your categories with the data your shop needs. You can also localize the each category property for your
```
Comment (pascal-zarrad) [inline docs/index.md]:
```suggestion
many more to enhance your categories with the data your shop needs. You can also localize each category property for your
```
I think the `the` does not belong there.

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+Install the package with Composer and the following command:
+
+``` bash
+composer require flagbit/category_bundle
+```
+
+#### Enable the bundle
+
+Enable the bundle in the kernel:
```
Comment (pascal-zarrad) [inline docs/index.md]: Maybe we can leave out this sentence as done under "Import the routing" as it looks weird in the documentation. Also the code example already contains all necessary information (file path, code to paste)

File: docs/custom-types.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+## Create custom property types
+
+While the Category Bundle comes with many build-in property types, your project might need one that is too specific to
+your use case than it could be added to the Category Bundle. Don't worry, it allows you to create your own property type
+that can be used in your project without contributing to the Bundle.
```
Comment (pascal-zarrad) [inline docs/custom-types.md]: Maybe we should rephrase this a bit, as the first sentence is hard to read.

Maybe something like?
```suggestion
While the Category Bundle comes with many build-in property types, your project might need one that is very specific to your use case. Adding such a specific type to the official bundle might not make sense. Don't worry, the bundle allows you to create your own property type that can be used in your project without contributing to the Bundle.
```

File: docs/custom-types.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    }
+}
+
+const factory: PropertyFactory = (): Property => new MyProperty();
+
+export default factory;
+```
+
+The implemented `render()` will get an instance of `PropertyDto` which also contains a method to register changes that
+occur when you do changes in your custom property on the category page. You can also access:
```
Comment (pascal-zarrad) [inline docs/custom-types.md]: This sentence is also really complex, consider rephrasing it.

**Review by pascal-zarrad — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 2 above)
Comment (flagbird) [reply docs/index.md]: Removed.

File: docs/custom-types.md (hunk 4 above)
Comment (flagbird) [reply docs/custom-types.md]: @pascal-zarrad What do you think about:

```suggestion
The implemented `render()` method will get an instance of `PropertyDto` injected. The `PropertyDto` contains a method to 
apply changes from your custom property on the category page. You can also access:
```

Comment (pascal-zarrad) [reply docs/custom-types.md]: @flagbird this would be fine 👍🏻

**Review by pascal-zarrad — APPROVED** [review]
LGTM 👍

**Pull request merged**